### ISSUE TITLE
python3

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [rafaelmartins/g-octave#10]
Is this program python3 ready?

**Comment by rafaelmartins** [comment]
No, I looked at porting it to python3 last week, but it should be a good amount of work to port. Don't know when I'll have time to do it.
